### DISCUSSION TITLE
Google telemetry in it???

### DISCUSSION THREAD

**Discussion by olafjorgen5685** [SkyTubeTeam/SkyTube#1019] [Q&A]
Tracker from OpenTelemetry?
https://reports.exodus-privacy.eu.org/en/reports/search/free.rm.skytube.oss/
https://reports.exodus-privacy.eu.org/en/trackers/412/
https://opencensus.io/
"OpenCensus originates from Google, where a set of libraries called Census were used to automatically capture traces and metrics from services."
https://opentelemetry.io/

DO NOT WANT

**Comment by gzsombor** [comment]
It seems the google-http-java-client depends on that library, there is a strict dependency as of now, and they have an open ticket to remove - https://github.com/googleapis/google-http-java-client/issues/621
There are only a couple of places, where SkyTube still uses the offical Youtube API - and that library, so my guess, it is possible to remove the whole library if you could implement that calls. Other solution could be to patch the forked [google-http-java-client](https://github.com/SkyTubeTeam/google-http-java-client) and use that.
